datastructures/src/Rhyolite/Map/Monoidal.hs

```diff
@@ -3,14 +3,16 @@
 {-# LANGUAGE StandaloneDeriving #-}
 module Rhyolite.Map.Monoidal (module X, (=:), restrictKeys) where
 
+import Control.Lens (At, Index, IxValue, set, at)
 import Data.AppendMap as X
 import Data.Map.Monoidal as X
 
 import qualified Data.Set as Set
 
 -- | Operator for creating a singleton 'Map'
```

Comment:
The docs are wrong now?

Reply:
Maybe the generalized operator doesn't "belong" to this module anymore? 
Since we could use it as well for the `=:` defined in  reflex-dom